### ISSUE TITLE
TypeError: TastytradeClient is not a constructor

### ISSUE THREAD

**Issue by m1771vw** [tastytrade/tastytrade-api-js#27]
Hi,

I'm following the quick start in the README:

```
import TastytradeClient from "@tastytrade/api"
const tastytradeClient = new TastytradeClient(baseUrl, accountStreamerUrl)
const loginResponse = await tastytradeClient.sessionService.login(usernameOrEmail, pasword)
```

I'm getting this error message:
```
const tastytradeClient = new TastytradeClient(baseUrl, accountStreamerUrl)
                         ^

TypeError: TastytradeClient is not a constructor
```

The baseUrl I'm using is the sandbox page.

I'm on `"@tastytrade/api": "^3.1.0",`
I'm on node v21.5
I'm also trying to incorporate it with express and use it in a route.

Let me know what other information you need

**Comment by ferdousbhai** [comment]
Experiencing the same issue.

**Comment by dmoss18** [comment]
@m1771vw Can I see your package.json file and (if you're using typescript) your tsconfig.json?

**Comment by osdevisnot** [comment]
@m1771vw and @ferdousbhai - are you able to create a smaller reproduction of your issue? Looking at the code example above, the usage of import indicates that you are using this in a project that supports modern import syntax. Ideally this should be fixed once #30 is merged and published. In case it does not, the smaller reproduction will help a lot in troubleshooting this problem.

**Comment by m1771vw** [comment]
@osdevisnot 

Package.json: 

```
{
	"name": "tasty-trade-api-playground",
	"version": "1.0.0",
	"description": "",
	"main": "src/index.js",
	"type": "module",
	"scripts": {
		"test": "echo \"Error: no test specified\" && exit 1",
		"start": "nodemon src/index.js"
	},
	"keywords": [],
	"author": "",
	"license": "ISC",
	"dependencies": {
		"@tastytrade/api": "^3.1.0",
		"axios": "^1.6.7",
		"express": "^4.18.2",
		"lodash": "^4.17.21",
		"nodemon": "^3.1.0"
	}
}

```

**Comment by osdevisnot** [comment]
Thanks for reporting @m1771vw! I can confirm the issue. The fix for this will be available as soon as we release #30 to NPM. Stay tuned! cc: @dmoss18
